Source/Scene/MapboxImageryProvider.js

```diff
@@ -17,8 +17,14 @@ define([
     'use strict';
 
     var trailingSlashRegex = /\/$/;
-    var defaultCredit1 = new Credit('© Mapbox © OpenStreetMap', undefined, 'https://www.mapbox.com/about/maps/');
-    var defaultCredit2 = [new Credit('Improve this map', undefined, 'https://www.mapbox.com/map-feedback/')];
+    var defaultCredit1 = new Credit({
+        text:'© Mapbox © OpenStreetMap',
+        link: 'https://www.mapbox.com/about/maps/'
+    });
+    var defaultCredit2 = [new Credit({
```

Comment:
Can `text` be HTML? If so, it might be better to merge this into a single credit so it's clear that the `Improve this map` credit is part of the mapbox layer.  Not a huge deal either way, just thought I'd bring it up..

Reply:
Nope, text can only be text right now